tests/unit/test_conf.py

```diff
@@ -22,6 +22,8 @@
 from contextlib import contextmanager
 import warnings
 
+import ssl
+import sys
 import pytest
```

Comment:
```suggestion
from contextlib import contextmanager
import ssl
import sys
import warnings

import pytest
```

There are usually three (or four) blocks of imports (each one alphabetically sorted) separated by a blank line:
 * standard lib packages (those that come shipped with Python)
 * third party packages (those you'd install with pip or similar)
 * local imports (importing package internal things)
 * (only for tests) importing the things under test
 
 The reason for this is:
 * Easier to read and find things if you stick to a consistent scheme
 * Should some import collide with another, you at least make sure that the most important things are last and won't be overwritten.